src/dotnet/commands/dotnet-tool/update/xlf/LocalizableStrings.de.xlf

```diff
@@ -23,8 +23,8 @@
         <note />
       </trans-unit>
       <trans-unit id="CommandDescription">
-        <source>Update a tool to the latest stable version.</source>
-        <target state="translated">Hiermit wird ein Tool auf die neueste stabile Version aktualisiert.</target>
+        <source>Update a global tool.</source>
```

Comment:
Same here.

Reply:
We don't have update in the costing. It is a good to have feature. I think we should keep it global tools only for now

Reply:
I didn't change here